docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -66,15 +66,15 @@ Including `--skip-pow-validation-enabled` skips validation of the `mixHash` when
 
 Exports a block or range of blocks from storage to a file in RLP format.
 
-If you omit `--start-block`, the start block defaults to 0 (the beginning of the chain), and if you
-omit `--end-block`, the end block defaults to the end of the chain.
+If you omit `--start-block`, the default start block is 0 (the beginning of the chain), and if you
+omit `--end-block`, the default end block is the end of the chain.
```

Comment:
nit: we usually would say "current chain head" rather than "end of the chain" 
not sure if you would want to make that change here - it might be in other places